modules/ROOT/pages/metrics-list.adoc

```diff
@@ -32,19 +32,20 @@ You can obtain them from applications, the Open Liberty runtime, and the Java vi
 For more information about building observability into your applications, see xref:microservice-observability-metrics.adoc[Microservice observability with metrics].
 
-When using MicroProfile Metrics with the feature:monitor-1.0[display=Performance Monitoring] feature, you enable the reporting of `vendor` metrics on the `/metrics` endpoint.
-Prior to MicroProfile Metrics 2.3, the feature:monitor-1.0[display=Performance Monitoring] feature needs to be explicitly configured.
-In MicroProfile Metrics 2.3 and up, the feature:monitor-1.0[display=Performance Monitoring] feature is automatically enabled by the MicroProfile Metrics feature during startup.
-It is worth noting that the _REST base metrics_ introduced in MicroProfile Metrics 2.3 also relies on the feature:monitor-1.0[display=Performance Monitoring] feature to function.
+To use the MicroProfile Metrics with the feature:monitor[display=Performance Monitoring] feature, enable the reporting of `vendor` metrics in the `/metrics` endpoint.
 
-The feature:monitor-1.0[display=Performance Monitoring] feature retrieves statistical data from all available monitoring components.
+Before MicroProfile Metrics 2.3, the Performance Monitoring feature had to be explicitly configured.
+From MicroProfile Metrics 2.3, the Performance Monitoring feature is automatically enabled by MicroProfile Metrics during startup.
+The _REST base metrics_ introduced in MicroProfile Metrics 2.3 also relies on the Performance Monitoring feature to function.
 
-You can identify which MicroProfile Metrics release the associated metrics are offered by identifying the relevant metric in the _Metrics that are available for Open Liberty_ table, and the version it was introduced in.
+The Performance Monitoring feature retrieves the statistical data from all available monitoring components.
+
+You can identify the relevant metric associated with the MicroProfile Metrics release in the <<#metrics-table,Metrics that are available for Open Liberty>> table, and the version it was introduced in.
 For example, the Request Timing vendor metrics were introduced in MicroProfile Metrics 2.0.
 
-By default, all monitoring components are enabled. You can configure the exact monitoring components to be enabled by configuring the `server.xml` with the `filter` attribute for the monitor configuration element.
+By default, all monitoring components are enabled. You can configure the exact monitoring components to be enabled by configuring the `server.xml` file with the `filter` attribute for the monitor configuration element.
 
-To enable only the monitoring components used by MicroProfile Metrics you can configure the `server.xml`, as shown in the following example:
+To enable only the monitoring components used by MicroProfile Metrics you can configure the `server.xml` file, as shown in the following example:
```

Comment:
If your server is collecting more metrics data than you need, you can improve server performance by collecting only those metrics that you intend to use.
To enable only the monitoring components that are used by MicroProfile Metrics,  you add the following code to your `server.xml` file.